build.gradle

```diff
@@ -346,6 +346,22 @@ task rsrdinttest(type: Test, dependsOn: 'jar') {
 	}
 }
 
+task msrdinttest(type: Test, dependsOn: 'jar') {
+    testClassesDir = sourceSets.slowtest.output.classesDir
+    classpath = sourceSets.slowtest.runtimeClasspath
+    forkEvery = 1
+    maxHeapSize = "512m"
+    include 'pcgen/inttest/game_modern/*Test.class'
+    testLogging {
+        exceptionFormat = 'full'
+    }
+
+	// Report each test as it is started.
```

Comment:
Probably a tab vs spaces. There was a noted inconsistency when I copied it from the rsrd version.